@vates/generator-toolbox/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "ES2024",
+    "module": "NodeNext",
+    "moduleResolution": "nodenext",
```

Comment:
There is a particular reason to not use the same config as this [PR](https://github.com/vatesfr/xen-orchestra/pull/8316/files#diff-b998a181f2fc52bd6811ea54e41e53acc5c512a735fa9694d167d89c3873fbeaR7)?
```
    "module": "ESNext",
    "moduleResolution": "bundler",
```